src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -507,6 +507,10 @@ impl Node {
         self.view.id().into()
     }
 
+    pub fn all_edges(self) -> Vec<EdgeId> {
```

Comment:
no docs, add them pls. I know this file is far from perfect, but lets add them here and there :D